feature-libs/my-account/organization/styles/__ux/_list.scss

```diff
@@ -379,3 +386,26 @@ cx-organization-form {
 //   }
 
 // }
+cx-unit-list {
+  @include media-breakpoint-down(md) {
+    th {
+      width: 0;
+      opacity: 0;
+      > ng-component {
+        width: 0;
+        opacity: 0;
+      }
+    }
+
+    tr.name {
+      a {
+        align-self: center;
+      }
+
+      button.button.action {
```

Comment:
lengthy selector. I'd like a more semantic selector, i.e. `toggle-tree` and `expand` or `collapse`.